src/main/webapp/i18n/de/exercise.json

```
@@ -45,6 +45,7 @@
             "problemStatement": "Aufgabenstellung",
             "sampleSolution": "Beispiel Lösung",
             "gradingInstructions": "Benotungshinweise",
+            "gradingInstruction": "Benotungshinweise: ",
```

```suggestion
            "gradingInstruction": "Benotungshinweis: ",
```
Otherwise it would also be plural

---

aaah, okay..